### DISCUSSION TITLE
pass arguments on execute async script

### DISCUSSION THREAD

**Discussion by drdedou** [seleniumbase/SeleniumBase#2511] [Q&A] [answered]
why i can't pass arguments on execute async script
sb.execute_async_script() 
(method) def execute_async_script(
    script: Any,
    timeout: Any | None = None
) -> Any

**Comment by mdmintz** [comment, marked as answer]
`execute_async_script()` comes from regular/raw Selenium: https://github.com/SeleniumHQ/selenium/blob/df7e464a6886abf7bb66a1694fc8370507ada262/py/selenium/webdriver/remote/webdriver.py#L409

So far, the only usage of that in SeleniumBase is here: (in a `wait_for_angularjs()` method, where the internal JS code was probably borrowed from Stack Overflow) https://github.com/seleniumbase/SeleniumBase/blob/c5169e55d1f1dc6cd89cede7bbe23a064787f1ec/seleniumbase/fixtures/js_utils.py#L54

A better place to ask about that method would be in https://github.com/SeleniumHQ/selenium or on https://stackoverflow.com/
